site/content/in-dev/unreleased/admin-tool.md

```diff
@@ -0,0 +1,157 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+linkTitle: Polaris Admin Tool
+title: Apache Polaris (incubating) Admin Tool
+type: docs
+weight: 300
+---
+
+In order to help administrators manage their Polaris database, Polaris provides an administration
+tool.
+
+The tool is built using [Quarkus](https://quarkus.io/).
+
+## How to Download the Admin Tool
+
+As of January 2025, there is currently no binary release or official Docker image available for
+the tool. For now, you need to build the artifacts yourself, for example, by running the following
+command:
```

Comment:
I think it makes sense to use a matched version for both polaris server and admin tool, even after the binary release. In that case, if a user/developer is using the main branch, they should compile the admin tool anyway.
```suggestion
Make sure the admin tool and Polaris server are with the same version. If you are using Polaris from the source code, you need to build the artifacts yourself by running the following command:
```

Reply:
I'll apply the suggestion, but for the record, while it is definitely better if server and tool have the exact same version, that is not a hard requirement. What needs to match is the metastore/database schema.